shub/apps/base/management/commands/show_settings.py

```diff
@@ -0,0 +1,31 @@
+"""
+
+Copyright 2017-2023 Evan Felix.
```

Comment:
When you contribute to an OSS project you typically have to use the same Copyright headers.

Reply:
I just took the header out, as it seems the MPL does not require it.